ObjectiveGit/GTCredential.h

```diff
@@ -46,7 +46,7 @@ NS_ASSUME_NONNULL_BEGIN
 /// type     - the credential types allowed by the operation.
 /// URL      - the URL the operation is authenticating against.
 /// userName - the user name provided by the operation. Can be nil, and might be ignored.
-- (GTCredential * _Nullable)credentialForType:(GTCredentialType)type URL:(NSString *)URL userName:(nullable NSString *)userName;
+- (GTCredential * _Nullable)credentialForType:(GTCredentialType)type URL:(nullable NSString *)URL userName:(nullable NSString *)userName;
```

Comment:
The libgit2 documentation implies that it's never NULL. At least `git_remote_connect` will return an error on a `NULL` URL. It seems the only other user is proxy support, and in that case it's not clear (there's a check skipping NULLs in `git_proxy_options_dup`).